en/docs/cookies/amp.md

```diff
@@ -0,0 +1,82 @@
+#  Automatic Mixed Precision Training
+
+## Introduction to AMP 
+
+When we train deep learning models, we typically use 32-bit single-precision floating point (FP32), while **AMP (Automatic Mixed Precision)** is a technique that allows both FP32 and FP16 to be used when training models. This can make the memory usage less and the calculation faster when training the model. But because the numerical range of FP16 is smaller than that of FP32, it is more prone to numerical overflow problems, and there may be some errors. But lots of practice has proved that many deep learning models can be trained with this technique without loss of accuracy.
```

Comment:
```suggestion
When we train deep learning models, we typically use 32-bit single-precision floating point (FP32), while **AMP (Automatic Mixed Precision)** is a technique that allows both FP32 and FP16 to be used when training models. This can make the memory usage less and the computation faster when training the model. But because the numerical range of FP16 is smaller than that of FP32, it is more prone to numerical overflow problems, and there may be some errors. But lots of practice has proved that many deep learning models can be trained with this technique without loss of accuracy.
```

Reply:
caculation 和 computation 都可以，不过好像 OneFlow 里统一用 computation 更多一点。